react-frontend/src/index.js

```diff
@@ -3,10 +3,14 @@ import ReactDOM from "react-dom";
 import "./index.css";
 import App from "./App";
 import "bootstrap/dist/css/bootstrap.min.css";
+import { Provider } from "react-redux";
+import configureStore from "redux-store";
```

Comment:
For consistency, follow this import order:

- core/important imports from third party modules
- other third party imports
- line break
- import custom modules
- line break
- import CSS modules